pkg/runtime/processor/pubsub/pubsub.go

```diff
@@ -238,6 +248,10 @@ func matchRoutingRule(rules []*rtpubsub.Rule, data map[string]interface{}) (*rtp
 	return nil, nil
 }
 
+func ExtractCloudEventProperty(cloudEvent map[string]any, property string) string {
+	return extractCloudEventProperty(cloudEvent, property)
```

Comment:
Why not just rename it to ExtraCloudProperty and avoid the wrapper?

Reply:
Renamed it. Originally wanted to avoid multiple changes.